torchvision/models/mobilenetv2.py

```diff
@@ -46,7 +46,7 @@ def __init__(
             norm_layer = nn.BatchNorm2d
         if activation_layer is None:
             activation_layer = nn.ReLU6
-        super(ConvBNReLU, self).__init__(
+        super(ConvBNActivation, self).__init__(
```

Comment:
Perhaps simplify using the Python3 idiom?
```suggestion
        super().__init__(
```